modules/scrabble.js

```diff
@@ -44,10 +46,19 @@ function scoreLetter (letter, str) {
     }
   }
 }
+function scrabbleNotate (str) {
+  var word = str.toLowerCase()
+  for (var letter of [...new Set(word)]) {
+    for (var i = 0; i < bag[letter]['count']; i++) {
```

Comment:
```suggestion
    for (let i = 0; i < bag[letter]['count']; i++) {
```

there's probably a better way of looping in the Array prototype somewhere but I'm not sure what